static/src/stylesheets/module/facia-garnett/item-layouts/_fc-item--full-media-100.scss

```diff
@@ -22,6 +22,12 @@ x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
 */
 
 @mixin fc-item--full-media-100 {
+
+    .responsive-img {
+        top: -25%;
```

Comment:
Why top `-25%`?

Reply:
This rule was originally specified in the original `facia` equivalent of `full-media-100.scss` (here's the original PR: https://github.com/guardian/frontend/commit/1efb05030f4e51f77b880ec3664ce80418bfc1b9)

It should've been copied over when the `facia-garnett` version was set-up, there's no reason why it should not be in this file.

Reply:
It's pulling a 5:3 crop into a 5:2 one by moving the image up so the top overflows and then uses padding and overflow:hidden on the parent div to give it vertical height.

Reply:
Same weird trick that is applied to showcased images in articles of tone Features (only) on desktop breakpoint. I’ve heard we are getting rid of that behaviour. Are we keeping this one then? (I have seen this card type maybe once in real life).

Reply:
Yeah, it's that trick again.  :(